close_pull_requests.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+"""
+    Close PRs on repositories where the master is not on github.
+
+    Provide a closing comment, and print the lock URL if desired
+"""
+
+import logging
+import argparse
+import yaml
+from client import get_github3_client
+
+DEFAULT_MESSAGE = 'Pull requests not accepted. Please see CONTRIBUTING.'
+DEFAULT_CONFIG = 'close_pull_requests.yaml'
+
+logger = logging.getLogger(__name__)
+exit_code = 0
+
+
+def update_exit_code(new_code):
```

Comment:
I'll r+ this shortly because there's no use nitpicking, but I'd probably comment here why this exists, it had me confused at first -- but you don't want to mask an error by resetting the exit code to 0 essentially, right